hip-service-test/Discovery/Builder/IdentifierBuilder.cs

```diff
@@ -0,0 +1,17 @@
+namespace hip_service_test.Discovery
+{
+    using HipLibrary.Patient.Models;
+
+    // ReSharper disable once ClassNeverInstantiated.Global
+    internal class IdentifierBuilder
+    {
+        private IdentifierType Type { get; set; }
+
+        private string Value { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private set accessor in property 'Value'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4780538)